tests/parse1.dora

```diff
@@ -1,6 +1,6 @@
 fun main() {
-  assert("1".parseInt() == 1);
-  assert("100".parseInt() == 100);
-  assert("789".parseInt() == 789);
-  assert("bla".parseInt() == 0);
+  assert("1".toInt().unwrap() == 1);
+  assert("100".toInt().unwrap() == 100);
+  assert("789".toInt().unwrap() == 789);
+  assert("bla".toInt().isNone());
```

Comment:
This doesn't work:

```
#> target/debug/dora tests/parse1.dora
uncaught exception
3: String.toIntOrThrow() -> Int: 27
2: String.toInt() -> Option[Int]: 30
1: main(): 5
```

Any idea why?

Reply:
I think `toIntOrThrow` and `toLongOrThrow` need to be marked as `throws`. I would assume that this is the problem here, could also be a bug though.

Reply:
Done, thanks!